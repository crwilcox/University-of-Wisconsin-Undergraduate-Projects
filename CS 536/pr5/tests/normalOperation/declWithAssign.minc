void main()
{
	int x = 4;
	int y = 5;
	double a = 4.0;
	double b = 5.5e4;
	bool d = true;
	return;
}
