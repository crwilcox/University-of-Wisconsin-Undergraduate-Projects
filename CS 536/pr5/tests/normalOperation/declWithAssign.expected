void main() {
  int x = 4;
  int y = 5;
  double a = 4.0;
  double b = 55000.0;
  bool d = true;
  return;
}
